hawkular-nest/hawkular-nest-itest-parent/hawkular-nest-itest/src/test/java/org/hawkular/nest/StatusEndpointITest.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2015 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.nest;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.jar.Attributes;
+import java.util.jar.Manifest;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipInputStream;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.jboss.shrinkwrap.resolver.api.maven.Maven;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.squareup.okhttp.OkHttpClient;
+import com.squareup.okhttp.Request;
+import com.squareup.okhttp.Response;
+
+/**
+ * @author <a href="https://github.com/ppalaga">Peter Palaga</a>
+ *
+ */
+@RunWith(Arquillian.class)
+public class StatusEndpointITest {
+
+    private static final String statusUrl = "http://127.0.0.1:8080/hawkular/nest/itest/status";
+    private static final String itestWarPath = System.getProperty("hawkular.nest.itest.war.path");
+
+    @Deployment(testable = false)
+    public static WebArchive createDeployment() {
```

Comment:
Just curious: why did you decide to have the status WAR in a separate maven module? There are only 4 files on this WAR (I think), so, it could have been simplified by building the WAR via Shrinkwrap as well.